src/operations/extrusions/slice/reverse.js

```diff
@@ -0,0 +1,25 @@
+const create = require('./create')
+
+/**
+ * Reverse the rotation of the given slice
```

Comment:
'rotation' alone might be a bit misleading ?
rotation of the points making up the edges perhaps ?

Reply:
Changed the doc strings.